src/french.rs

```diff
@@ -0,0 +1,135 @@
+use itertools::Itertools;
+
+static ONES: [&str; 10] = [
+    "zéro", "un", "deux", "trois", "quatre", "cinq", "six", "sept", "huit", "neuf",
+];
+
+static TEENS: [&str; 10] = [
+    "dix",
+    "onze",
+    "douze",
+    "treize",
+    "quatorze",
+    "quinze",
+    "seize",
+    "dix-sept",
+    "dix-huit",
+    "dix-neuf",
+];
+
+static TENS: [&str; 8] = [
+    "vingt", "trente", "quarante", "cinquante", "soixante", "soixante-dix", "quatre-vingt", "quatre-vingt-dix",
+];
+
+static BASES: [&str; 6] = [
+    "mille",
+    "million",
+    "milliard",
+    "billion",
+    "billiard",
+    "trillion",
+];
+
+static BASES_MULT: [&str; 6] = [
```

Comment:
It is not very clear what `BASES_MULT` means. Perhaps `BASES_PLURAL` is a better name here?